### PR TITLE
Log rotation hinted

### PR DESCRIPTION
**Pull request by earfolds**
The test test_wp_metrics_format currently always passes.

All other tests should be working.

### CODE REVIEW

File: learner-instructions/LO-13-log-rotation.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Log rotation
+
+You have been asked to deploy WordPress once again, and to create log files to keep track of system metrics over time. You will need to **adhere to a strict format**, to **schedule the logs to update once a minute** and to **ensure the log files stay a manageable size**.
+
+Your tasks today are as follows:
+
+ - **Deploy WordPress** in the ```/var/www/html/wordpress/``` directory.
+ - **Schedule a log** in a **system-wide cron job** in the ```/etc/cron.d/wp-metrics``` file that appends the current date and time in ```YYYY-MM-DD HH:mm``` format to the ```/var/log/wp-metrics/system.log``` file once every minute 
+   - After the date and time, include the current disk usage of the ```/var/www/html/wordpress/``` directory in bytes
```
Comment (JamesBelchamber) [inline learner-instructions/LO-13-log-rotation.md:9]: "..as well as the directory path"

File: learner-instructions/LO-13-log-rotation.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+     - For example, ```2022-09-14 09:32 16777216 /var/www/html/wordpress```
+ - Use the **logrotate** utility and a configuration file at ```/etc/logrotate.d/wp-metrics``` to switch to new log files once daily, and to archive 7 completed days of logs.
+ - Finally, do the same thing to keep track of the ***current CPU and memory usage** of the **root httpd process** in ```/var/log/wp-metrics/httpd.log```
```
Comment (JamesBelchamber) [inline learner-instructions/LO-13-log-rotation.md:12]: The same thing? Maybe number the steps so you can refer to them explicitly.

**Review by JamesBelchamber — CHANGES_REQUESTED** [review]
A few tidy-ups, looking good :+1:

File: learner-instructions/LO-13-log-rotation.md (hunk 1 above)
Comment (earfolds) [reply learner-instructions/LO-13-log-rotation.md]: Resolved by https://github.com/generation-org/tech-foundations-labs/pull/13/commits/03ef9d9c3a9555401c4a37b31021955053bcd1b2

File: learner-instructions/LO-13-log-rotation.md (hunk 2 above)
Comment (earfolds) [reply learner-instructions/LO-13-log-rotation.md]: Resolved by https://github.com/generation-org/tech-foundations-labs/pull/13/commits/03ef9d9c3a9555401c4a37b31021955053bcd1b2

**Pull request merged**